player.py

```diff
@@ -32,19 +32,26 @@ def make_move(self, state):
 
         pawn = None
         valid = False
-        print('Begin by selecting a pawn in a row and column')
-        while not valid:
-            row, col = eval(input('row, col: '))
-            pawn = state.get_pawn(row, col, self.color)
-            if pawn is not None:
-                valid = True
         final_pos = None
-        valid = False
-        print('Select a final position for the pawn')
         while not valid:
-            row, col = eval(input('row, col: '))
-            if state.check_cell(row, col) == label.BLANK:
-                final_pos = Position(row, col)
-                valid = True
+            print('Begin by selecting a pawn in a row and column')
+            row = eval(input('Select a row: '))
+            col = eval(input('Select a column: '))
+            if (row in range(3)) and (col in range(3)):
```

Comment:
Although the range is small, the `range(...)` function allocates memory, which is a slow operation overall. Maybe we can simplify with interval comparisons?

```python
if 1 <= row <= 3 and 1 <= col <= 3:
```